docs/getting-started-with-snowplow-bdp/aws-setup-guide/index.md

```diff
@@ -64,32 +64,37 @@ You will need to share this role with us as part of filling out the setup form i
  "cloudwatch:*",
  "dynamodb:*",
  "ec2:*",
- "es:*",
+ "ecs:*",
+ "eks:*",
  "elasticbeanstalk:*",
  "elasticloadbalancing:*",
  "elasticmapreduce:*",
+ "es:",
```

Comment:
Is this missing a `*`?

Reply:
Yes there was :( - thanks for spotting.
Added and tidied a couple of space issues.